### PR TITLE
support for the G E-Chip weather station protocol

### PR DESCRIPTION
**Pull request by prutsky**
Added support for GT-WT-01 weather station sensor.
This time in the development branch, based on the plight development branche.

### CODE REVIEW

**Comment by CurlyMoo** [comment]
Again, do PR's according to the pilight contribution guide.

File: libs/pilight/protocols/433.92/ge_chip.c [hunk 1]
```diff
@@ -0,0 +1,207 @@
+/*
+	Copyright (C) 2014 CurlyMo
```
Comment (CurlyMoo) [inline libs/pilight/protocols/433.92/ge_chip.c]: Please add your own name here.

**Comment by CurlyMoo** [comment]
What was the problem with the channel code?

**Comment by prutsky** [comment]
I couldn't get the channel to show up in the GUI. Local version decode okay. Temperature and humidity are fine for now.

File: libs/pilight/protocols/433.92/ge_chip.c [hunk 2]
```diff
@@ -0,0 +1,207 @@
+/*
+	Copyright (C) 2015 Prutsky
```
Comment (CurlyMoo) [inline libs/pilight/protocols/433.92/ge_chip.c]: I said add, not replace mine :stuck_out_tongue_winking_eye:

**Comment by CurlyMoo** [comment]
Also, even though the channel doesn't show in the GUI, we want as complete support as we can.

**Comment by prutsky** [comment]
get latest DEV

**Pull request closed without merging**